docs/advanced-features/debugging.md

```diff
@@ -4,74 +4,89 @@ description: Debug your Next.js app.
 
 # Debugging
 
-This documentation explains how you can debug your Next.js frontend and backend code with full source maps support using either the [Chrome DevTools](https://developers.google.com/web/tools/chrome-devtools) or the [VSCode debugger](https://code.visualstudio.com/docs/editor/debugging).
+This documentation explains how you can debug your Next.js front-end and back-end code with full source maps support using either the [VS Code debugger](https://code.visualstudio.com/docs/editor/debugging) or [Chrome DevTools](https://developers.google.com/web/tools/chrome-devtools).
```

Comment:
Lets stick with `frontend` and `backend` https://english.stackexchange.com/a/34943/9506